### ISSUE TITLE
.NET standard 2.0

### ISSUE THREAD

**Issue by pdufrene** [dotnet/docs#22011]
A sample of csproj targeting netstandard 2.0 will be welcome:
I tried the following csproj:
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
    <EnableNETAnalyzers>true</EnableNETAnalyzers>
  </PropertyGroup>

</Project>
```
But all rules keep their default severity, looks like the AnalysisMode is ignored, it works properly if I do:
`<TargetFramework>net5.0</TargetFramework>`

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f8906b2c-b984-0427-d064-829e7ea2fc55
* Version Independent ID: 128f9903-971d-6a66-a2cb-9b12acbc55e7
* Content: [Code analysis in .NET](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview#code-style-analysis)
* Content Source: [docs/fundamentals/code-analysis/overview.md](https://github.com/dotnet/docs/blob/master/docs/fundamentals/code-analysis/overview.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
@mavasani This should work without having to change the target framework to `net5.0`, correct?

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
    <EnableNETAnalyzers>true</EnableNETAnalyzers>
  </PropertyGroup>

</Project>
```

**Comment by mavasani** [comment]
Few things to note:

1. The above settings do not work in .NET 5.0 SDK due to a known bug. You can workaround the bug replacing  `<EnableNETAnalyzers>true</EnableNETAnalyzers>` with `<AnalysisLevel>latest</AnalysisLevel>`
2. Above bug has been fixed for the upcoming 5.0.2xxx release of the SDK. See https://github.com/dotnet/sdk/pull/14767 for details.

**Comment by gewarren** [comment]
Thanks Manish. I'm going to close the issue since the doc will be correct once 5.0.2xx ships.
